src/container/container.ts

```diff
@@ -579,7 +588,7 @@ class Container implements interfaces.Container {
   ): (T | T[]) {
     const result = this._get<T>(getArgs);
 
-    if (isPromiseOrContainsPromise<T>(result)) {
+    if (!this.options.allowLazyInSync && isPromiseOrContainsPromise<T>(result)) {
       throw new Error(ERROR_MSGS.LAZY_IN_SYNC(getArgs.serviceIdentifier));
```

Comment:
`getArgs.serviceIdentifier` can be a symbol here, and it fails to get converted to string in `LAZY_IN_SYNC`.

I added a quick fix in `error_msgs.ts` to not fail in such a case. Not sure if the impl is good but it's there.